Common/src/fem_geometry_structure.cpp

```diff
@@ -340,7 +340,11 @@ CMeshFEM::CMeshFEM(CGeometry *geometry, CConfig *config) {
   /*--- and must be generalized later. As this number is also needed   ---*/
   /*--- in the solver, it is worthwhile to create a separate function  ---*/
   /*--- for this purpose.                                              ---*/
-  const bool compressible = (config->GetKind_Regime() == COMPRESSIBLE);
+  unsigned short Kind_Solver = config->GetKind_Solver();
+  const bool compressible = (Kind_Solver == FEM_EULER) ||
+                            (Kind_Solver == FEM_NAVIER_STOKES) ||
```

Comment:
My comment is still valid I suppose.

Reply:
Which one? I reverted most of the GetRegimeType calls. I deliberately left it in some places to make clear for what solvers this part of the code is intended for. For the future we have to find a better way of getting properties for a certain kind of solver.